bigbluebutton-html5/imports/startup/mapToAcl.js

```diff
@@ -0,0 +1,39 @@
+import Acl from '/imports/startup/acl';
+import { Meteor } from 'meteor/meteor';
+import Logger from '/imports/startup/server/logger';
+
+const emptyCollection = new Mongo.Collection("emptyCollection");
```

Comment:
> When you return an array (in a publication), Meteor loops through the array of cursors and sends an added document to the client for each element in the cursors. When it's done it then sends the ready event to the client. That's why return []; does sends the ready notification, because it loops through the empty array, it doesn't send any records to the client and then fires the ready event.
...
**And thus this.ready() is better than return []; which is better than return People.find( { "_id": "-1" });**